### PR TITLE
Css styles for main page, flowers page and components css fixes

### PR DESCRIPTION
**Pull request by Arvydasss**

### CODE REVIEW

File: frontend/src/components/cards/Card.tsx [hunk 1]
```diff
@@ -6,19 +6,26 @@ import "./Card.scss";
...
             <Card.Img variant="top" src="./yesterday.png" />
             <Card.Body>
-                <Card.Title>{title}</Card.Title>
-                <Card.Text>
+                <Card.Title className="card-title">{title}</Card.Title>
+                <Card.Text className="card-description">
                     {description}
                 </Card.Text>
-                <Button text="Add To Cart" />
+                    <Button text={"Add To Cart"} className="btn-add-to-cart" />
```
Comment (medikornov) [inline frontend/src/components/cards/Card.tsx:26]: whitespace

File: frontend/src/components/header/Header.tsx [hunk 2]
```diff
@@ -31,10 +31,10 @@ export const Header = () => {
     return (
         <div className="header">
             <div className="header-logo">
-                <Button text={"Bucket Flowers"} onClick={() => navigate("/")} />
+                <Button text={"Bucket flowers"} onClick={() => navigate("/")} className='btn-home' />
```
Comment (medikornov) [inline frontend/src/components/header/Header.tsx:34]: don't like the idea of using this class here

**Review by medikornov — APPROVED** [review]

File: frontend/src/components/header/Header.tsx (hunk 2 above)
Comment (Arvydasss) [reply frontend/src/components/header/Header.tsx:34]: why? both buttons have almost equal styles, i just modify it to look more like an icon, not a button

Comment (medikornov) [reply frontend/src/components/header/Header.tsx:34]: Its about the usage of this class.. The `btn-home` class is not ours, so in the future there might be issues with that. For example after package update.

**Review by 0rl1s — APPROVED** [review]

**Pull request merged**